csaf_2.1/prose/edit/src/distributing-04-transition-20-21.md

```diff
@@ -0,0 +1,74 @@
+## Transition between CSAF 2.0 and CSAF 2.1
+
+This subsection details the process that SHOULD be followed when transitioning the distribution of documents from CSAF 2.0 to CSAF 2.1.
+Different scenarios can be encountered:
+
+- Providers will continue to use CSAF 2.0.
+  This SHOULD be avoided, as the CSAF ecosystem greatly profits from adoption to the new standard version.
+- Providers will immediately upgrade all documents as well as the `provider-metadata.json` to CSAF 2.1.
+  While this benefits the adoption of CSAF 2.1, consumers that still rely on CSAF 2.0 are cut off.
+- Providers will begin a transition period, in which they continue to serve existing documents in CSAF 2.0, gradually updating the existing
+  document base (e.g. by using a CSAF 2.0 to CSAF 2.1 converter as described in [sec](#conformance-clause-18-csaf-2-0-to-csaf-2-1-converter))
+  and publishing new documents using CSAF 2.1.
+
+### Announcing the Transition
+
+In the last scenario, a temporary parallel distribution of CSAF 2.0 and CSAF 2.1 documents and provider metadata is recommended.
+The provider SHOULD announce a transition period containing three points in time:
+
+- The begin of the transition period, where the provider is starting to serve CSAF 2.1 documents, while CSAF 2.0 being authoritative.
+- The roll-over-date at which CSAF 2.1 becomes authoritative but CSAF 2.0 is still supported.
+- The end of the transition period, after which CSAF 2.0 is not supported any more.
+
+The announcement MAY contain also the following information:
+
+- The first date, when CSAF 2.1 documents are available and automatically retrievable from the server through a CSAF 2.1 `provider-metadata.json`.
+- The last date, when CSAF 2.0 documents are available and automatically retrievable from the server through a CSAF 2.0 `provider-metadata.json`.
```

Comment:
Let's clarify that as well.
```suggestion
- The last date, when CSAF 2.0 documents are available and automatically retrievable from the server through a CSAF 2.0 `provider-metadata.json`.
  > This date is usually at the end or after the transition period.

Reply:
I thought this markes the end of the transition period as we have defined it now. I would not include the "after the" part.